src/jit/codegenxarch.cpp

```diff
@@ -1412,6 +1412,221 @@ void CodeGen::genCodeForBinary(GenTree* treeNode)
     genProduceReg(treeNode);
 }
 
+// Deals with codegen for muti-register struct returns.
```

Comment:
Also, could you add a line not to indicate what has changed? It is difficult to tell now that you have extracted this functionality into a separate method (a good thing to do, but also one of the reasons it is usually prudent to separate refactoring changes from functional changes).